src/components/SharedComponents/VideoPlayer/VideoPlayer.tsx

```diff
@@ -39,6 +39,13 @@ export type VideoPlayerProps = {
   title: string;
   location: VideoLocationValueType;
   isLegacy: boolean;
+  userEventCallback?: (event: VideoEventCallbackArgs) => void;
```

Comment:
its worth taking this approach given the many use cases for the video player. The callback will also be helpful later in triggering video related prompts